numba_dppy/dpnp_glue/dpnp_array_creations_impl.py

```diff
@@ -127,7 +127,7 @@ def dpnp_ones_like_impl(a, dtype=None):
 
     """
     res_dtype = dtype
-    if dtype == types.none:
+    if dtype == types.none or dtype is None:
```

Comment:
Why are these changes in this PR? Are the related to the @vectorize work?

Reply:
They are not, but they were discovered during the work somehow.